CONTRIBUTING.md

```diff
@@ -86,10 +86,37 @@ component of the Thanos distributed system. We recommend:
 `make help` will print most of available commands with relevant details.
 * Spin up a prebuilt dev environment using Gitpod.io
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/thanos-io/thanos)
+* In case you want to develop the project locally, install **Golang** in your machine. Here is a nice [gist](https://gist.github.com/nikhita/432436d570b89cab172dcf2894465753) for this purpose. -
```

Comment:
```suggestion
* In case you want to develop the project locally, install **Golang** in your machine. Here is a nice [gist](https://gist.github.com/nikhita/432436d570b89cab172dcf2894465753) for this purpose.
```